streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewUntypedDemo.java

```diff
@@ -99,7 +99,8 @@ public JsonNode apply(JsonNode view, String region) {
                         return new KeyValue<>(viewRegion.get("region").textValue(), viewRegion);
                     }
                 })
-                .countByKey(TimeWindows.of("GeoPageViewsWindow", 7 * 24 * 60 * 60 * 1000L).advanceBy(1000), Serdes.String())
+                .groupByKey(Serdes.String(), jsonSerde)
```

Comment:
It is because the KGroupedStream needs the Serdes so they need to be passed along somewhere (KStreamImpl doesn't have a reference to them)